tests/react-dom/render.test.js

```diff
@@ -0,0 +1,145 @@
+import { describe, test, expect } from "vitest";
+import { createElement, Fragment } from "../../src/lib/jsx";
+import { ReactDOM } from "../../src/lib/react-dom";
+
+describe("render Unit Test", () => {
+  test("render 함수는 파라미터로 Virtual DOM 객체와 Dom 객체를 받아 실제 DOM 에 반영한다.", () => {
+    const vdom = createElement(
+      "div",
+      { id: "app" },
+      createElement("p", null, "Hello World")
+    );
+
+    document.body.innerHTML = `<div id="root"></div>`;
```

Comment:
p4: 아래처럼 미리 선언하면 중복코드 꽤나 없앨 수 있습니당
```suggestion
  beforeEach(() => {
    document.body.innerHTML = `<div id="root"></div>`;
  });```